pydicom/filereader.py

```diff
@@ -886,6 +892,11 @@ def read_dicomdir(filename="DICOMDIR"):
 
     This is a wrapper around :func:`dcmread` which gives a default file name.
 
+    .. deprecated:: 2.1
+
+        ``read_dicomdir()`` is deprecated and will be removed in v2.2. Use
+        :func:`~pydicom.filereader.dcmread` instead.
```

Comment:
Still should be v3.0 here.

Reply:
Should it? Everyone should be using `dcmread()` by this point.

Reply:
Perhaps, but I'm pretty much fully on-board now with "no breaking changes" (except for minor things with very good reasons) unless a major version change.  And removal of a function could certainly break old code.  I think that philosophy might also help with #1137 (which I'm about to work on btw), by keeping more things aligned to the same version change.

Reply:
Fair enough